src/main/java/com/mraof/minestuck/client/gui/StoneTabletScreen.java

```diff
@@ -105,7 +105,7 @@ protected void init() {
 	@Override
 	public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTicks)
 	{
-		this.renderBackground(poseStack);
+		this.renderBackground(poseStack); //TODO issue on this line, "java.lang.NullPointerException: Cannot read field "level" because "this.minecraft" is null"
```

Comment:
The comment is insufficient to identify or recreate the issue. Where is the full crash log? Or is this just meant as a reminder for yourself before this PR gets merged?

Reply:
I shared the context in the server, so since the problem has been noted there I will remove it here